R/calc_fit_mod.R

```diff
@@ -0,0 +1,43 @@
+#' @title Calculates fitted and residual values for modeled outputs
+#' @description
+#'  `calc_fit_mod()` takes antibody kinetic parameter estimates and calculates
+#'  fitted and residual values. Fitted values correspond to the estimated assay
+#'  value (ex. ELISA units etc.) at time since infection (TSI). Residual values
+#'  are calculate as the difference between fitted and observed values.
+#' @param input_dat A [] dataframe of modeled antibody kinetic parameter values.
+#' @param original_data A [] data frame of the original input dataset.
+#' @returns A [data.frame] attached as an [attributes] with the following
+#' values:
+#'   - Subject = ID number specifying an individual
+#'   - Iso_type = The modeled antigen_isotype
+#'   - t = Time since infection 
+#'   - fitted = The fitted value calculated using model output parameters for a
+#'   given `t`
+#'   - residual = The residual value calculated as the difference between
+#'   observed and fitted values for a given `t`
+
```

Comment:
I think this change will fix the `pkgdown` problem:

```suggestion
#' @keywords internal
```